rpc/client/httpclient.go

```diff
@@ -339,13 +339,15 @@ func (c *baseRPCClient) Tx(hash []byte, prove bool) (*ctypes.ResultTx, error) {
 	return result, nil
 }
 
-func (c *baseRPCClient) TxSearch(query string, prove bool, page, perPage int) (*ctypes.ResultTxSearch, error) {
+func (c *baseRPCClient) TxSearch(query string, prove bool, page, perPage int, sortOrder string) (
+	*ctypes.ResultTxSearch, error) {
 	result := new(ctypes.ResultTxSearch)
 	params := map[string]interface{}{
-		"query":    query,
-		"prove":    prove,
-		"page":     page,
-		"per_page": perPage,
+		"query":      query,
+		"prove":      prove,
+		"page":       page,
+		"per_page":   perPage,
+		"sort_order": sortOrder,
```

Comment:
why `sort_order` and not simply `sort`?

Reply:
Actually Inside function everywhere I user sortOrder because we can't use `sort` as a variable name because we are using `sort` as a module. That's why to keep consistency I have user `sort_order` Instead of the `sort`. But, If you think `sort` is better then, I can make the changes.

Reply:
oh, sort module, right. how about `orderBy`?

Reply:
Yes `orderBy` sounds good but `sort_order` is also good. Should I change it to `order_by`? @melekes

Reply:
for context: https://www.postgresql.org/docs/8.1/queries-order.html where order_by comes from

Reply:
I like `order_by`. 🚲 🏠

Reply:
or maybe simply `order` will work

Reply:
Wait, I thought "order by" typically just refers to asc or desc? In SQL, for example.

Reply:
Done @melekes @tessr